### PR TITLE
docs: Show hidden sites and fix broken links

### PR DESCRIPTION
**Pull request by Asiderr**

### CODE REVIEW

File: docs/variants/protectli_ptx01/overview.md [hunk 1]
```diff
@@ -22,8 +22,5 @@ TPM.
   open-source firmware update methods.
 * [Recovery](recovery.md) - gathers information on how to recover the platform
   from potential failure.
-* [Hardware configuration matrix](hardware-matrix.md) - describes the
-  platform's hardware configuration used during the Dasharo firmware
-  validation procedure.
-* [Test matrix](test-matrix.md) - describes validation scope used during
-  Dasharo firmware validation procedure.
```
Comment (miczyg1) [inline docs/variants/protectli_ptx01/overview.md:29]: @Asiderr we need those two files in the current form. Please revert

File: docs/variants/protectli_ptx01/releases.md [hunk 2]
```diff
@@ -15,3 +15,6 @@ Test results for this platform can be found
 [here](https://docs.google.com/spreadsheets/d/1wI0qBSLdaluayYsm_lIa9iJ9LnPnCOZ9eNOyrKSc-j4/edit#gid=1316498194).
 
 [newsletter]: https://newsletter.3mdeb.com/subscription/n2EpSxtqL
+
+<!--Empty pixel to avoid orphaned pages when overview is hidden-->
+[![empty-pixel](../../images/empty_pixel.png)](overview.md)
```
Comment (miczyg1) [inline docs/variants/protectli_ptx01/releases.md:20]: Not sure i understand what is it for... Why other files do not need it?

File: mkdocs.yml [hunk 3]
```diff
@@ -78,12 +78,6 @@ plugins:
         - variants/server.md
         - variants/protectli_ptx01/hardware-matrix.md
         - variants/protectli_ptx01/test-matrix.md
```
Comment (miczyg1) [inline mkdocs.yml:80]: We have to restore those two as well

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: docs/variants/protectli_ptx01/releases.md (hunk 2 above)
Comment (Asiderr) [reply docs/variants/protectli_ptx01/releases.md:20]: @miczyg1 We do not have other local links to `overview.md` when it is hidden and for this reason it becomes an orphaned page. Adding an empty pixel with a link to `overview.md` fixes this problem.

File: mkdocs.yml (hunk 3 above)
Comment (Asiderr) [reply mkdocs.yml]: Fixed fbacd307

File: docs/variants/protectli_ptx01/overview.md (hunk 1 above)
Comment (Asiderr) [reply docs/variants/protectli_ptx01/overview.md]: Fixed fbacd307

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**